src/mongocrypt.h.in

```diff
@@ -149,7 +149,8 @@ typedef enum {
    MONGOCRYPT_STATUS_OK = 0,
    MONGOCRYPT_STATUS_ERROR_CLIENT = 1,
    MONGOCRYPT_STATUS_ERROR_KMS = 2,
-   MONGOCRYPT_STATUS_ERROR_CSFLE = 3,
+   MONGOCRYPT_STATUS_ERROR_CRYPT_SHARED = 3,
+   MONGOCRYPT_STATUS_ERROR_CSFLE = MONGOCRYPT_STATUS_ERROR_CRYPT_SHARED,
```

Comment:
> There is a single enumerator name that should be changed, but deleting it would be an API change. I'm not sure that there's anyone actually using that name, though?

Searching all drivers suggests no use of `MONGOCRYPT_STATUS_ERROR_CSFLE`. It is not included in a stable release of libmongocrypt. I suggest removing.

```
$ ls all_drivers                              
clone_all.sh            mongo-cxx-driver        mongo-php-driver        mongo-ruby-driver       node-mongodb-native
mongo-c-driver          mongo-go-driver         mongo-php-library       mongo-rust-driver       update_all.sh
mongo-csharp-driver     mongo-java-driver       mongo-python-driver     mongo-swift-driver
$ ag MONGOCRYPT_STATUS_ERROR_CSFLE all_drivers
```